src/ui/albumcovermanager.cpp

```diff
@@ -424,6 +424,7 @@ bool AlbumCoverManager::ShouldHide(const QListWidgetItem& item,
 }
 
 void AlbumCoverManager::FetchAlbumCovers() {
+  cover_fetcher_->fetchall_ = true;
```

Comment:
This shouldn't be a member variable at all but a parameter to the cover fetch method, possibly with a default.

Reply:
That's what I intended in the first place, but confused somewhere. I've redone it now, I'm creating a variable on the request instead, this allows automatically downloading while playing from discogs even when the user could be doing fetch all missing covers in the cover manager.